windows/src/engine/keyman32/kmprocess.cpp

```diff
@@ -549,18 +551,14 @@ int PostString(PWSTR str, LPMSG mp, LPKEYBOARD lpkb, PWSTR endstr)
 			  _td->app->QueueAction(QIT_BELL, 0);
 			  break;
 		  case CODE_CONTEXT:				// copy the context to the output
-			  for(q = _td->miniContext; *q; q++) {
-			    _td->app->QueueAction(QIT_CHAR, *q);
-        }
-			  break;
+        PostString(_td->miniContext, mp, lpkb, wcschr(_td->miniContext, 0));
```

Comment:
Does the Windows version of `PostString()` not have a corresponding parameter `ignoreOutputKeystroke`?

https://github.com/keymanapp/keyman/blob/91b0795f1d2321c40ec197efac67742889fe3809/common/core/desktop/src/kmx/kmx_processor.cpp#L361

Reply:
The Windows version has a global which is equivalent. The core version is the new one which will hopefully replace the Windows one before too long!